lib/AST/ASTImporter.cpp

```diff
@@ -2628,6 +2609,16 @@ Decl *ASTNodeImporter::VisitFunctionDecl(FunctionDecl *D) {
   const FunctionDecl *FoundByLookup = nullptr;
   FunctionTemplateDecl *FromFT = D->getDescribedFunctionTemplate();
 
+  // Check if we have found an existing definition.  Returns with that
+  // definition if yes, otherwise returns null.
+  auto IsDefinitionFound = [this](
+      FunctionDecl *D, FunctionDecl *FoundFunction) -> Decl * {
```

Comment:
`FindAndMapDefinition`?
`D` can be captured in the lambda, or call the parameter `FromFunction` instead.

Reply:
Ok, I like the name you suggest.